apps/website/src/pages/docs/datepicker.mdx

```diff
@@ -22,36 +22,64 @@ import * as AllExamples from '~/examples';
 
 When a user needs to schedule deadlines, timeframes, or for age verification purposes, we provide them with a date picker. It behaves much like a select menu. We currently have three different types of date pickers.
 
-## Appearance
+## Variants
 
-### Default
+### Basic
 
-The default date picker is a default menu button from which an interactive calendar expands on click. This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters.
+The basic date picker consists of a default menu button from which an interactive calendar expands on click.
 
-### Text field
+<LiveExample src='DatePicker.basic.tsx' truncate={false}>
+  <AllExamples.DatePickerBasicExample client:load />
+</LiveExample>
+
+This allows the user to visually select the date rather than entering the date with a keyboard. This has the advantage of also showing weekdays, which may be important in a given context. The date picker is commonly used within column filters.
+
+### Localized
+
+You are able to create a localized version of the date picker that uses local names for week days and months.
+
+<LiveExample src='DatePicker.localized.tsx' truncate={false}>
+  <AllExamples.DatePickerLocalizedExample client:load />
+</LiveExample>
+
+### Date + time selection
 
-There are two ways to select a date with the text field date picker; clicking on the button and selecting a date, or typing it out using the format shown in the help message underneath the field (DD-MM-YYYY). The date automatically fills the text field when directly selected in the calendar and is editable. It’s a more convenient way to offer editing in a context where the user might need to change the date often, or enter different dates for several items in a workflow.
+If you need to include the time, there are multiple ways to do so. Below is the default variant, which uses twenty-four hour / military time.
```

Comment:
Remove "military-time", I think 24-hour format is enough.

Reply:
Fixed.